cypress/utils/utils.ts

```diff
@@ -1212,15 +1214,19 @@ export function selectUserPerspective(userType: string): void {
 }
 
 export function selectWithinModal(selector: string): void {
-    cy.get(modal).within(() => {
-        click(selector);
-    });
+    cy.get(modal)
+        .eq(0)
+        .within(() => {
+            click(selector);
+        });
```

Comment:
**nitpick:** This function is a copy of `clickWithin` so it can be modified to call that function:

    export function selectWithinModal(selector: string): void {
        clickWithin(modal, selector);
    }

Reply:
Oh, I see. Will make changes in `selectWithinModal` and will still keep two different functions.

Reply:
Done. Thanks !